src/Resources/app/administration/src/module/findologic-module/components/findologic-config/index.js

```diff
@@ -229,16 +194,15 @@ Component.register('findologic-config', {
       return this.repositoryFactory.create('category');
     },
 
-    categoryCriteria () {
-      const criteria = new Criteria(1, 500);
-      criteria.addSorting(Criteria.sort('name', 'ASC'));
-      criteria.addSorting(Criteria.sort('parentId', 'ASC'));
+    categories () {
+      return this.categoryCollection ?? [];
```

Comment:
Why does this even exist? Can't `this.categoryCollection` already be `[]` by default?

Reply:
I am not sure about this, I am just using the logic from Shopware

Reply:
I adapted it, and it seems to still work